bn-apps/cordapp-updates-distribution/corda-updates-transport/src/main/kotlin/net/corda/cordaupdates/transport/CordaTransporterFactory.kt

```diff
@@ -0,0 +1,27 @@
+package net.corda.cordaupdates.transport
+
+import org.eclipse.aether.RepositorySystemSession
+import org.eclipse.aether.repository.RemoteRepository
+import org.eclipse.aether.spi.connector.transport.Transporter
+import org.eclipse.aether.spi.connector.transport.TransporterFactory
+import org.eclipse.aether.transfer.NoTransporterException
+
+/**
+ * Implementation of Maven Resolver's [TransporterFactory] for Corda transport.
+ *
+ * Corda transport expects a repository URL to be specified in the format of "corda:x500Name", i.e. for example "corda:O=BNO,L=New York,C=US"
```

Comment:
Maybe we can extend it to include some notion of "repository name" as well as just legal name, so one node can host multiple independent repositories? e.g. on behalf of others?

Reply:
Great idea. Added. So now it's `corda:x500Name$repoName`.